### ISSUE TITLE
Logger posts text/plain requests with ISO-8859-1 encoding

### ISSUE THREAD

**Issue by dsagomonov** [sswayney/seq-logback-appender#13]
By default, org.apache.http.entity.StringEntity constructs requests body with text/plain mime-type and ISO-8859-1 encoding. That's why SEQ shows international symbols as '?'. I think that using application/json mime-type and UTF-8 encoding will be better.

**Comment by sswayney** [comment]
@dsagomonov, I reviewed your PR #14 for the fix. It was Merged and well go out in next release. Waiting on another PR. Thank you.

**Comment by sswayney** [comment]
@dsagomonov  Fixed in version 1.0.1 Thank you for your contribution. 👍
